roles/transformers/tasks/main.yml

```diff
@@ -13,59 +35,18 @@
   poll: 0
   loop: "{{ utils_transform }}"
 
-- name: Enable EPEL repository
-  yum_repository:
-    name: epel
-    description: EPEL YUM repo
-    metalink: https://mirrors.fedoraproject.org/metalink?repo=epel-{{ ansible_distribution_major_version }}&arch=$basearch&infra=$infra&content=$contentdir
-    gpgcheck: yes
-    gpgkey: https://dl.fedoraproject.org/pub/epel/RPM-GPG-KEY-EPEL-{{ ansible_distribution_major_version }}
-
-- name: Download imagemagick-distribution-linux.rpm
-  get_url:
-    url: "{{ imagemagick_rpm }}"
-    checksum: sha1:{{ imagemagick_sha1_checksum }}
-    dest: "{{ download_location }}/imagemagick-distribution-linux.rpm"
-    mode: 0444
-    url_username: "{{ nexus_user }}"
-    url_password: "{{ nexus_password }}"
-    timeout: 570
-
-- name: Download imagemagick-distribution-libs-linux.rpm
-  get_url:
-    url: "{{ imagemagick_lib_rpm }}"
-    checksum: sha1:{{ imagemagick_lib_sha1_checksum }}
-    dest: "{{ download_location }}/imagemagick-distribution-libs-linux.rpm"
-    mode: 0444
-    url_username: "{{ nexus_user }}"
-    url_password: "{{ nexus_password }}"
-    timeout: 570
-
-- name: install imagemagick
-  yum:
-    name: "{{ download_location }}/{{ item }}"
-    state: present
-    disable_gpg_check: true
-  loop:
-    - imagemagick-distribution-libs-linux.rpm
-    - imagemagick-distribution-linux.rpm
-
-- name: "Download libreoffice {{ libreoffice }} rpm tar.gz"
-  get_url:
-    url: "{{ libreoffice_rpm }}"
-    checksum: sha1:{{ libreoffice_sha1_checksum }}
-    dest: "{{ download_location }}/libreoffice.tar.gz"
-    url_username: "{{ nexus_user }}"
-    url_password: "{{ nexus_password }}"
-    mode: 0444
-    timeout: 950
-  register: libreoffice_download
-  async: 900
-  poll: 0
+- name: Prepare RPM based systems
+  when: pkg_fmt != 'deb'
+  block:
+    - name: Enable EPEL repository
+      yum_repository:
```

Comment:
I've seen that in other places you've handled dnf also. why not here?

Reply:
essentially because dnf just wraps yum. I think places where dnf is handled specifically are tasks that are specific to dnf?